Lesson1/Selim_Osmanov/HT1_3(forward_list)/HT1_3(forward_list)/test/HT1_3gtest.cpp

```diff
@@ -0,0 +1,66 @@
+#include <forward_list>
+#include <vector>
+#include "gmock/gmock.h"
+
+template <typename T>
+void ReverseElements (std::forward_list <T> & MyForwardList) {
```

Comment:
why you copying the implementation to the tests? what if you'll test a 3k+ lines file?